plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/repository/MavenCoordinatesHelper.java

```diff
@@ -44,33 +44,42 @@
   private static final String CLASSPATH_ATTRIBUTE_CLASSIFIER =
       "com.google.cloud.tools.eclipse.appengine.libraries.classifier";
 
-  public List<IClasspathAttribute> createClasspathAttributes(Artifact artifact,
-                                                                            MavenCoordinates mavenCoordinates) {
+  private MavenCoordinatesHelper() { }
+
+  public static List<IClasspathAttribute> createClasspathAttributes(MavenCoordinates mavenCoordinates,
+                                                             String actualVersion) {
```

Comment:
indentation fixed, however I think line wrapping would decrease readability for the sake of not having 2 characters over the 100 char limit.